apps/discord/.env.example

```
@@ -1,5 +1,8 @@
 MODAL_IMAGE_BUILDER_VERSION=2025.06
 
+SUPABASE_URL=YOUR_SUPABASE_URL_HERE
+SUPABASE_SERVICE_ROLE_KEY=YOUR_SERVICE_ROLE_KEY_HERE
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The SUPABASE_SERVICE_ROLE_KEY key should go before the SUPABASE_URL key

(UnorderedKey)